code-quality/code-validation.js

```diff
@@ -23,7 +23,7 @@ var codeValidationFunctions = {
         //name: sausage-metric
 
         this.testName = "Valid script name prefix";
-        this.reason = "Prefixes is important not only to distinguish which type of device the script is executed on but also to avoid script name collisions";
+        this.reason = "Prefixes are important, not only to distinguish which type of device the script is executed on, but also to avoid script name collisions.";
```

Comment:
Thanks, I do these all the time. English is my second language so I appreciate all the help I can get. :)